src/prefs/DevicePrefs.cpp

```diff
@@ -415,12 +415,16 @@ bool DevicePrefs::Commit()
    return true;
 }
 
-namespace{
-PrefsPanel::Registration sAttachment{ "Device",
-   [](wxWindow *parent, wxWindowID winid, AudacityProject *)
-   {
+PrefsPanel::Factory DevicePrefsFactory()
+{
+   return [](wxWindow *parent, wxWindowID winid, AudacityProject *) {
```

Comment:
You might make this simpler than the corresponding code in SpectrumPrefs.  There, I needed a non-empty lambda capture.  Here, you don't.  So just declare instead

```
PrefsPanel *DevicePrefsFactory(wxWindow *parent, wxWindowID winid, AudacityProject *)
```